R/xml_find.R

```diff
@@ -62,23 +62,23 @@
 #' ')
 #' xml_find_all(x, ".//f:doc")
 #' xml_find_all(x, ".//f:doc", xml_ns(x))
-xml_find_all <- function(x, xpath, ns = character()) {
+xml_find_all <- function(x, xpath, ns = xml_ns(xml_root(x))) {
```

Comment:
I think this would be better as `ns = xml_ns(x)`, i.e. the logic for finding the root should be in `xml_ns()` (so we should probably make it generic)